test/widgets/clean_app_bar_test.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:collaction_app/presentation/shared_widgets/custom_app_bars/clean_app_bar.dart';
```

Comment:
Can we format test files a bit better? Just add `,` wherever it's possible almost, feel the indentation is completely off for both test files! :D